services/s3_gate/docker-compose.yml

```diff
@@ -17,14 +17,12 @@ services:
       - ./tls.key:/tls.key
       - ./tls.crt:/tls.crt
       - ./../../vendor/hosts:/etc/hosts
+      - ./cfg:/etc/frostfs/s3
     stop_signal: SIGKILL
     env_file: [ ".env", ".s3.env", ".int_test.env" ]
+    command: [ "frostfs-s3-gw", "--config", "/etc/frostfs/s3/config.yml" ]
     environment:
-      - S3_GW_WALLET_PATH=/wallet.json
-      - S3_GW_WALLET_PASSPHRASE=s3
       - S3_GW_RPC_ENDPOINT=http://morph-chain.${LOCAL_DOMAIN}:30333
-      - S3_GW_SERVER_0_TLS_KEY_FILE=/tls.key
-      - S3_GW_SERVER_0_TLS_CERT_FILE=/tls.crt
```

Comment:
Why we dropped this here but not added to `config.yml`?

Reply:
do not remember exactly. mb mistake. but on the other hand is it expected to work if not [`enabled`](https://github.com/nspcc-dev/neofs-s3-gw/blob/50d85dc7edabe6a753c346c388bf18bf9134cd90/config/config.yaml#L31) config is provided?

Reply:
added